README.md

```diff
@@ -34,8 +33,10 @@ Here's how to use it in a playbook:
 - hosts: all
   become: yes
   become_method: sudo
-  roles:
-    - amazon-efs-utils
+  tasks:
+    - name: Install Amazon EFS utilities
+      ansible.builtin.include_role:
+        name: amazon-efs-utils
```

Comment:
Hyphens aren't valid right? Also should we instead match what's in `meta/main.yml` (`efs_utils`)? Alternatively should `meta/main.yml` be updated to match this value?

```suggestion
        name: amazon_efs_utils
```

Reply:
Please see commit e736883.